app/src/main/java/com/android/shelfLife/ui/newoverview/NewOverview.kt

```diff
@@ -71,83 +56,73 @@ fun OverviewScreen(
   val scope = rememberCoroutineScope()
 
   HouseHoldSelectionDrawer(
-      scope = scope,
-      drawerState = drawerState,
-      navigationActions = navigationActions,
-      houseHoldRepository = houseHoldRepository,
-      userRepository = userRepository,
-  ) {
-    val filteredFoodItems =
-        foodItems.filter { item ->
-          item.foodFacts.name.contains(searchQuery, ignoreCase = true) &&
-              (selectedFilters.isEmpty() || selectedFilters.contains(item.foodFacts.category.name))
-        }
+      scope = scope, drawerState = drawerState, navigationActions = navigationActions) {
+        val filteredFoodItems =
+            foodItems.filter { item ->
+              item.foodFacts.name.contains(searchQuery, ignoreCase = true) &&
+                  (selectedFilters.isEmpty() ||
+                      selectedFilters.contains(item.foodFacts.category.name))
+            }
 
-    if (!householdViewModelIsLoaded) {
-      Column(
-          modifier = Modifier.fillMaxSize(),
-          horizontalAlignment = Alignment.CenterHorizontally,
-          verticalArrangement = Arrangement.Center,
-      ) {
-        CircularProgressIndicator()
```

Comment:
I feel like removing this is not the best idea, just in case something ever goes wrong when it comes to displaying the overview, I think having a CircularProgressIndicator is wise to have as a filler screen.